tests/frontend/travis/remote_runner.js

```diff
@@ -52,7 +52,8 @@ var sauceTestWorker = async.queue(function (testSettings, callback) {
 
     var knownConsoleText = "";
     var getStatusInterval = setInterval(function(){
-      browser.eval("$('#console').text()", function(err, consoleText){
+      var testResultFormatedScript = "function customReportFromHTML(){var globalResult='';var report=$('#report').clone();report.find('pre').remove();report.find('> li').each(function(index,element){var titleText=$(element).find('> h1').text();var testResultTitle=$(element).find('> ul > li > h1').text();var result='';$(element).find('> ul > li > ul > li').each(function(index,element){if($(element).hasClass('pass'))result+='[green]PASSED[clear] : ';if($(element).hasClass('fail'))result+='[red]FAILED[clear] : ';if($(element).hasClass('pending'))result+='[yellow]PENDING[clear] : ';result+=$(element).text()});globalResult+=titleText+''+testResultTitle+'  -> '+result+'  '});return globalResult;};customReportFromHTML();";
```

Comment:
🐙  ``\`` is a thing for line breaks to make this readable, worth using?

Reply:
@JohnMcLear, I have tried to use the line breaks as the original implementation, cause the runner handle this input to split in line breaks, but somehow using inside `eval` is throwing an error, I will try to figure out a way to add the line breaks properly in this case.

Reply:
Don't worry too much about it :)  Just a readability thing and this is not the worst none-line-broken piece of code in Etherpad ;)

Reply:
@JohnMcLear but the remote runner split the tests by these line breaks:
https://github.com/ether/etherpad-lite/blob/develop/tests/frontend/travis/remote_runner.js#L38

Reply:
If I can't find a solution I will figure out other separators...

Reply:
Ahhhhhh...  Okay that makes sense..  Did you try to throw the lines at remote runne rto see what happens yet?   I assume you implemented the sauce keys I sent you?

Reply:
If you are working on this now I will pause the debug I'm doing as they will interfere..

Reply:
Hey @JohnMcLear, yeah, I'm working on this and I got the sauce API keys and I'm able to run the tests on CI 🙏